addons/spreadsheet_dashboard_hr_expense/data/dashboards.xml

```diff
@@ -4,6 +4,8 @@
     <record id="spreadsheet_dashboard_expense" model="spreadsheet.dashboard">
         <field name="name">Expenses</field>
         <field name="data" type="base64" file="spreadsheet_dashboard_hr_expense/data/files/expense_dashboard.json"/>
+        <field name="spreadsheet_snapshot" eval="False"/>
+        <field name="spreadsheet_revision_ids" eval="[Command.clear()]"/>
```

Comment:
not sure about clearing *all* revisions

Reply:
ponderate between:
1. losing all modifications user did forever but no TB
2. potentially corrupted revisions that require a support intervention since the error is not well handled ATM

I think the first one is less problematic and tbh. As long as this disappears in 16.1, the only people would should update are those affected by the bug, which cannot edit the dashboard anyway

Reply:
> which cannot edit the dashboard anyway

You have a point... actually half a point because they can still dismiss the traceback and edit (I think)